### PR TITLE
Modernization of site

### PR DESCRIPTION
**Pull request by iamjolly**
Update Jekyll, USWDS, main nav and footer to meet the Digital Council's modernization checklist.

### CODE REVIEW

File: _includes/components/header.html [hunk 1]
```diff
@@ -88,32 +97,88 @@
...
+                type="hidden"
+                value="{{ site.search_site_handle }}" />
+              <div role="search">
+                <label for="query" class="usa-sr-only">Enter Search Term(s):</label>
+                <input
+                  autocomplete="off"
+                  class="usa-input usagov-search-autocomplete"
+                  id="query"
+                  name="query"
+                  type="search" />
```
[the remark this answers is not in the document]
Comment (iamjolly) [reply _includes/components/header.html:122]: Nope. There's just one `label` for the search `input`. Why do I only get false-positives from this bot?

File: _includes/components/header.html [hunk 2]
```diff
@@ -88,32 +97,88 @@
...
+                value="{{ site.search_site_handle }}" />
+                <div role="search">
+                  <label
+                    for="query" class="usa-sr-only">Enter search terms</label>
+                  <input
+                    autocomplete="off"
+                    class="usa-input usagov-search-autocomplete"
+                    id="query"
+                    name="query"
+                    type="search" />
```
[the remark this answers is not in the document]
Comment (iamjolly) [reply _includes/components/header.html:172]: Nope. There's just one `label` for the search `input`. Why do I only get false-positives from this bot? Also, it has a typo in it. I think it is also a poor redundancy to `axe-cli` we're running via circleci.

**Review by igorkorenfeld — COMMENTED** [review]
I think this looks good! Any chance you have a Federalist preview I can take a look at?

**Comment by iamjolly** [comment]
@igorkorenfeld - I've been wondering why there is no Federalist build happening here. Does `circleci` need to trigger a Federalist build if certain conditions are met, or do you think there's an issue where Federalist should run no matter what but somehow isn't?

FWIW, `circleci` is running `axe-cli` latest version and it is complaining about a "best practice" accessibility thing, which is something I'll need to raise upstream with USWDS. It shouldn't be a show-stopper issue, though.

**Comment by igorkorenfeld** [comment]
I think it should trigger any time there is a commit on GitHub. Since this is a fork though, you would have had to have added your forked repo to Federalist in order for it to watch it. I am guessing that's what might be missing?

**Comment by iamjolly** [comment]
@igorkorenfeld - The webhook for Federalist needed an update, so it's building now and available at: https://federalist-86ac2a7d-ee18-4863-bd98-a59377759671.app.cloud.gov/preview/18f/accessibility/iamjolly/modernization/

🌻 🌻 🌻

**Comment by igorkorenfeld** [comment]
Thanks for adding the Federalist preview! 

The only thing that caught my eye is the top level nav. It currently has all external links, except for home. I think this might be an unexpected pattern/behavior. An idea for an alternative: what if there was a second top-level page on the site called something like "Other Resources", which had a list of these links?

**Comment by iamjolly** [comment]
I appreciate that concern and like that suggestion. I'll make a page for related gov resources.

**Comment by iamjolly** [comment]
@igorkorenfeld - Updated the primary nav to be home, other gov accessibility sites, and more 18f guides... with the other gov accessibility sites being a dedicated page on this site. 

We'll need to do some content clean-up for consistency related to that, but I think that's best done in other branches.

**Comment by igorkorenfeld** [comment]
This is great, thanks Robert! Agree about the other clean ups, I think this is good to merge.

In terms of the other branches/future edits, one thing to think about is maybe hiding the sidenav on the "Other .Gov Accessibility Sites", since the sidenav really only corresponds to "Home" in the information hierarchy.

**Pull request merged**